fastai/tabular/core.py

```diff
@@ -270,7 +270,8 @@ def decodes(self, to:Tabular):
 @Normalize
 def setups(self, to:Tabular):
     store_attr(means=dict(getattr(to, 'train', to).conts.mean()),
-               stds=dict(getattr(to, 'train', to).conts.std(ddof=0)+1e-7))
+               stds=dict(getattr(to, 'train', to).conts.std(ddof=0)+1e-7),
+               but='to')
```

Comment:
Move the `but=`s to the first line of the function - no need to use xtra vertical space for that little thing! :)